src/sidebar/components/test/annotation-test.js

```diff
@@ -18,6 +18,16 @@ var fakeDocumentMeta = {
 };
 
 /**
+ * Return an `Error` object matching the type of error produced by failed API
+ * calls.
+ */
+function apiError(status, message) {
```

Comment:
`<annotation>` only cares about the `message` property of the `Error`, not about any other details - and that's part of the `Error` interface.

I actually agree it would be preferable to subclass error so that callers can test that the error came from the API. Unfortunately subclassing `Error` is a bit fiddly in JS (at least pre native ES6). I'd like to make the improvement here first and look at that separately.